shaky/src/main/java/com/linkedin/android/shaky/ShakeDelegate.java

```diff
@@ -87,12 +88,12 @@ public void setSensitivityLevel(@SensitivityLevel int sensitivityLevel) {
      * submission, such as user data, app version, etc.
      */
     @WorkerThread
-    public void collectData(Activity activity, Result data) {
+    public void collectData(@NonNull Activity activity, @NonNull Result data) {
     }
 
     /**
      * Called when the user submits the Feedback form. Creates and starts an email Intent.
      * This method can be overridden to send data to a custom URL endpoint, etc.
      */
-    public abstract void submit(Activity activity, Result result);
+    public abstract void submit(@NonNull Activity activity, @NonNull Result result);
```

Comment:
Can you also change the consumer side and add the nullability check for activity?
Unfortunately, activity Shaky.java is not guaranteed to be nonnull.